src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
-// write code here
+const wall = document.querySelector('.wall');
```

Comment:
It's a good practice to check if the element exists in the DOM before using it. This will prevent any potential errors if the element is not present or the selector is incorrect. You can use an if statement to check if `wall` and `spider` are not null before using them.